src/software/simulated_tests/validation_functions/one_meter_from_ball_validation.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include "software/simulated_tests/validation/validation_function.h"
+#include "software/world/world.h"
+
+void oneMeterFromBall(std::shared_ptr<World> world_ptr,
```

Comment:
I don't recall any rules where we need to be one meter from the ball. What will this be used for?